src/start_menu/main_settings/ChangeFontWeight.h

```diff
@@ -0,0 +1,3 @@
+#pragma once
+
+void Weight();//Declares the Weight() function to prompt the user for a weight choice and customize text weight
```

Comment:
Change the function's name to match the name of the file.

Also, see how comments for functions are done in other header files (with /***/ before the declaration)